src/main.rs

```diff
@@ -558,7 +658,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     kub_controller
         .deploy_secret(bootstrap_validator.secret())
         .await?;
-    info!("Deployed Bootstrap Secret");
+    info!("Bootstrap Secret deployed");
```

Comment:
micro-nit: I actually prefer the new active style more that you're using in the other logs, but it's up to you
```suggestion
    info!("Deployed Bootstrap Secret");
```

Reply:
ahh ya i went back and forth on this. will update